Connects the Google Contact Form

maxskewes opened the pull request:
Puts a, <iframe> tag to a google contact form

Closes issue #604

nrrao commented:
@maxskewes, I am seeing form not being center.
![screencapture-localhost-3000-about-contact-2021-07-29-22_08_07](https://user-images.githubusercontent.com/25930687/127589273-d4fc9dd7-b000-4b8c-a6df-112b057f7de5.png)

nrrao reviewed (commented):
@maxskewes, I am seeing form not being center.
![screencapture-localhost-3000-about-contact-2021-07-29-22_08_07](https://user-images.githubusercontent.com/25930687/127589273-d4fc9dd7-b000-4b8c-a6df-112b057f7de5.png)

maxskewes commented:
@nrrao Thanks for catching that. I centered it.

mariastudnicka commented:
@maxskewes thanks! does this mean we're able to upload an image?

nrrao reviewed (changes requested):
@maxskewes, The form is center now. Since there is commented code should we make a separate file before merging? 
Also,
frameborder='0'
marginheight='0'
marginwidth='0' should be frameBorder,marginHeight,marginWidth to avoid console warning messages.
iframe should have title property to avoid warning message. 
<iframe ... title="contactForm"> frame body </iframe>

maxskewes commented:
I have updated the PR. I removed the commented code from index.js, created a separate ContactForm component to be used when we are no longer using the Google form, corrected code for the iframe tag.

maxskewes commented:
@mariastudnicka The uploading of the image is taking place on a different page and is not part of this pull request.

nrrao reviewed (approved):
Looks Good.
